sdk/contentsafety/azure-ai-contentsafety/pom.xml

```diff
@@ -43,7 +43,9 @@
   </developers>
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <jacoco.skip>true</jacoco.skip>
+    <jacoco.min.linecoverage>0.4</jacoco.min.linecoverage>
+    <jacoco.min.branchcoverage>0.3</jacoco.min.branchcoverage>
+    <jacoco.skip>false</jacoco.skip>
```

Comment:
why do we skip jacoco here? This is GA lib, I think we should have test coverage.